lib/unobtainium/support/util.rb

```diff
@@ -38,6 +37,34 @@ def normalize_label(label)
         end
         return nil
       end
+
+      def clean_chrome_args(options)
+        optionscopy = ::Collapsium::Config::Configuration.new(options)
+        %i[desired_capabilities caps].each do |index|
+          begin
+            optionscopy[index]["chromeOptions"]["args"].uniq!
```

Comment:
oh I thought the ``uniq`` method (without the ``!``) made a new copy of the hash and you wouldn't have to care about this yourself.